### PR TITLE
Add min/max/step/prefix/suffix to ui.number

### PR DESCRIPTION
**Pull request by falkoschindler**
Following the discussion #765 this PR extends the `ui.number` element with additional parameters.
On blur the value is sanitized using the min and max values as well as the format string - if given.

### CODE REVIEW

**Review by rodja — CHANGES_REQUESTED** [review]

File: tests/test_number.py [hunk 1]
```diff
@@ -16,3 +16,16 @@ def test_apply_format_on_blur(screen: Screen):
...
+    ui.number('Number', min=0, max=10, value=5)
+    ui.button('Button')
+
+    screen.open('/')
+    screen.should_contain_input('5')
+
+    element = screen.selenium.find_element(By.XPATH, '//*[@aria-label="Number"]')
+    element.send_keys('6')
+    screen.click('Button')
+    screen.should_contain_input('10')
```
[the remark this answers is not in the document]
Comment (falkoschindler) [reply tests/test_number.py:31]: The value is already "5". After sending the key "6", the value is now "56", which is sanitized down to "10".

[the remark this answers is not in the document]
Comment (rodja) [reply tests/test_number.py:31]: Ah, I understand. It just appends. Maybe we should type "600" to make it more clear that the value is capped to 10?

[the remark this answers is not in the document]
Comment (falkoschindler) [reply tests/test_number.py:31]: Yes, we could do that. On the other hand the other number test using pi (3.1416) also only makes sense for _appending_ digits.

[the remark this answers is not in the document]
Comment (rodja) [reply tests/test_number.py:31]: Ok. I'm convinced. We leave it as it is ;-)

**Review by rodja — APPROVED** [review]

**Pull request merged**